doc/ide.md

```diff
@@ -70,4 +75,24 @@ make compile-commands-hotspot
 
 ### IDE support for Java code
 
-This section is a work in progress.
\ No newline at end of file
+#### IntelliJ IDEA
+
+The JDK project has a script that can be used for indexing the project
+with IntelliJ. After configuring and building the JDK, IntelliJ workspace
+can be generated using:
+
+```shell
+bash bin/idea.sh
+```
+
+This creates a subfolder called `.idea` in the current folder. To use it,
```

Comment:
I might consider rewording this.  Typically this is run at the top of the your openjdk fork/clone.